moped-toolbox/component_geo_processing/settings.py

```diff
@@ -42,6 +42,11 @@
     query ProjectComponentsQuery {
         moped_project(where: {is_deleted: {_eq: false}}) {
             project_id
+            moped_proj_tags(where: {is_deleted: {_eq: false}}) {
+                moped_tag {
+                    name
+                }
+            }
```

Comment:
tags are not available in the `project_list_view` yet - so i patched them into this part of the query.

it's a goofy script - but can be simplified later after we overhaul components schema

Reply:
Mike's PR adds tags to the project_list_view, but this works since we will overhauling things not too long from now